### DISCUSSION TITLE
Where can I find the plugins like AutoToolTip and AutoResize ?

### DISCUSSION THREAD

**Discussion by stixoffire** [6pac/SlickGrid#685] [Q&A] [answered]
Where can I find the plugins like AutoToolTip and AutoResize ?

I have clicked on the plugins and 3rd party not there , I have looked in the examples for a link to them and not there.

Anybody have a link to them , and why can't there be a link in the examples whether in the code or the example page - anywhere , somewhere ..?

**Comment by 6pac** [comment]
The examples are here: https://github.com/6pac/SlickGrid/wiki/Examples

There are two tooltip examples in the  'Plugins/Controls' section (search the page), and an auto-resize example at the end of the 'Layout' section: http://6pac.github.io/SlickGrid/examples/example-size-to-content.html

Note that the you can also set GridAutosizeColsMode to LegacyForceFit, which is a lot simpler to implement, but in my opinion doesn not give as good results.

**Reply by stixoffire** [reply]
is the AutoToolTip plugin .js file there ? I did not see it there or see a link to it ...  is it in NPM ?

**Comment by 6pac** [comment, marked as answer]
yes, it's referenced in http://6pac.github.io/SlickGrid/examples/example-autotooltips.html

the file itself is in ```plugins/slick.autotooltips.js```

**Reply by stixoffire** [reply]
I was looking for it because my ide was barking about it not existing. I checked the project folders and I do see it .. but now I nee dot figure out what the issue is for VS.. i

**Reply by 6pac** [reply]
Weird! I use VS myself, but on Windows and using ASP NET Core mvc, it's in the wwwroot folder. Never had any problems. Your setup might be completely different though!
